README.md

```diff
@@ -20,30 +20,82 @@ First create a conda environment and install relevant dependencies:
 conda create -y -n rescript
 conda activate rescript
 conda install \
-  -c conda-forge -c bioconda -c qiime2 -c https://packages.qiime2.org/qiime2/2023.5/tested/ -c defaults \
-  qiime2 q2cli q2templates q2-types q2-longitudinal q2-feature-classifier 'q2-types-genomics>2023.2' \
-  "pandas>=0.25.3" xmltodict ncbi-datasets-pylib
+  -c conda-forge -c bioconda -c qiime2 \
+  -c https://packages.qiime2.org/qiime2/2024.2/amplicon/tested/ \
+  -c defaults qiime2 q2cli q2templates q2-types q2-longitudinal q2-feature-classifier \
+  'q2-types-genomics>2023.2' "pandas>=0.25.3" xmltodict ncbi-datasets-pylib
 ```
 Install source:
 
 ```
 pip install git+https://github.com/bokulich-lab/RESCRIPt.git
+
+qiime rescript --help
 ```
 
-### Option 2: Install within QIIME 2 environment
-First activate your QIIME 2 environment (ver 2022.8 or later) and install relevant dependencies:
+### Option 2: Install within QIIME 2 environments `2024.2` and later.
+First activate your QIIME 2 environment and install relevant dependencies. 
+The typical apporoach will be something like the command below. *Make sure the qiime version within the 
+http string matches the version of the active qiime environment.*
 
 ```
-conda activate qiime2-2023.2
-conda install -c conda-forge -c bioconda -c qiime2 -c https://packages.qiime2.org/qiime2/2023.5/tested/ -c defaults \
-  xmltodict 'q2-types-genomics>2023.2' ncbi-datasets-pylib
+conda activate qiime2-amplicon-2024.2
+conda install -c conda-forge -c bioconda -c qiime2 \
+    -c https://packages.qiime2.org/qiime2/2024.2/amplicon/tested/ \
```

Comment:
```suggestion
    -c https://packages.qiime2.org/qiime2/2024.2/amplicon/passed/ \
```
We've changed our package dir structure slightly - so the closest equivalent to the tested channel is now the passed channel. That being said, using the released channel will also work since this is essentially just a wholesale install of the latest amplicon release.